src/app/shared/components/sidemenu/sidemenu.component.spec.ts

```
@@ -228,13 +228,13 @@ describe('SidemenuComponent', () => {
       {
-        title: 'Team expense reports',
+        title: 'Expense reports',
         isVisible:
           component.allowedActions.allowedReportsActions && component.allowedActions.allowedReportsActions.approve,
         route: ['/', 'enterprise', 'team_reports'],
       },
       {
-        title: 'Team advances',
+        title: 'Advances',
         isVisible:
           component.allowedActions.allowedAdvancesActions && component.allowedActions.allowedAdvancesActions.approve,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)